client/src/components/requester_task/partial-item.html

```diff
@@ -1,40 +1,49 @@
 <div id="requester-task-item">
   <div class="title list-group-item">Pick item</div>
-  <form>
+  <form name="item">
     <div class="list-group-item">
       <div class="form-group">
         <label for="product-name">I want</label>
-        <input type="text" class="form-control" id="product-name" placeholder="Product name" ng-model="ctrl.item">
+        <input name="product" type="text" class="form-control" id="product-name" placeholder="Product name" ng-model="ctrl.item" required>
+      </div>
+      <div class="validation-alert" ng-messages="item.product.$error" ng-show="item.product.$error && item.product.$touched">
+        <div ng-messages-include="./src/shared/validation/messages.html"></div>
       </div>
       <div class="form-group">
         <label for="additional-details">Details</label>
-        <input type="text" class="form-control" id="additional-details" placeholder="Additional details if necessary..." ng-model="ctrl.details">
+        <input name="additional-details" type="text" class="form-control" id="additional-details" placeholder="Additional details if necessary..." ng-model="ctrl.details">
       </div>
       <!-- <button class="btn btn-green" type="submit" ng-click="ctrl.setItem()">Confirm item</button> -->
     </div>
   </form>
-  <div class="title list-group-item">Pick Time and Price</div>
-  <form>
+  <div class="title list-group-item">Pick Time and additional</div>

```

Comment:
good catch... that was a mistake. I think i replaced the word "price" when i shouldnt have